services/keycloak/startup-scripts/00-configure-lagoon.sh

```diff
@@ -1959,6 +1959,110 @@ EOF
 EOF
 }
 
+function update_env_var_view_permissions {
+  CLIENT_ID=$(/opt/jboss/keycloak/bin/kcadm.sh get -r lagoon clients?clientId=api --config $CONFIG_PATH | jq -r '.[0]["id"]')
+  view_value_env_var_dev_environment=$(/opt/jboss/keycloak/bin/kcadm.sh get -r lagoon clients/$CLIENT_ID/authz/resource-server/permission?name=View+Environment+Variable+Value+for+Development+Environment --config $CONFIG_PATH | jq -r '.[0]["id"]')
+  view_value_env_var_prod_environment=$(/opt/jboss/keycloak/bin/kcadm.sh get -r lagoon clients/$CLIENT_ID/authz/resource-server/permission?name=View+Environment+Variable+Value+for+Production+Environment --config $CONFIG_PATH | jq -r '.[0]["id"]')
+  view_value_env_var_project=$(/opt/jboss/keycloak/bin/kcadm.sh get -r lagoon clients/$CLIENT_ID/authz/resource-server/permission?name=View+Environment+Variable+Value+for+Project --config $CONFIG_PATH | jq -r '.[0]["id"]')
+
+  if [ "$view_value_env_var_dev_environment" != null ]; then
+      echo "environment:viewValue:development on env_var already configured"
+      return 0
+  fi
+
+  echo Recreating \"View Environment Variable for Development Environment\" permissions
```

Comment:
Should move these echo statements from here to above the sections that the events take place, just so the logs reflect what is happening as they happen.